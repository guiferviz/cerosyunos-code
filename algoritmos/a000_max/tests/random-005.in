30
100
1868121526 1074616382 1182061662 1501224365 900719877 1500755844 512540339 1082399571 1500048153 829173011 1664279734 149643280 1123011832 597369486 1141250794 160446905 169391727 992635542 424615682 213589529 1378257051 229350829 48357810 866296721 1442225022 1140236097 1387763147 1468295158 178160226 1391944869 2114018070 1986841432 901681472 2145717105 737473745 558684342 1368629381 1386592512 139006664 1074100960 25091805 470468389 1818149048 426397759 1402807775 1543986470 394673496 1510761793 1338467631 831302104 1008363031 1244913327 685736437 1198128984 1084068781 1563355766 1215339068 941057599 1891425443 1071457462 690783239 2082672658 101264804 448208093 901546108 990095937 1926449428 1880478271 88872698 916233461 90066978 1209417984 1094983281 368579840 1980148561 1339931727 1830390077 502590300 44616863 1510984370 457412469 990143526 1227420746 1885707621 198023126 151394342 1395358135 1255025080 1263209855 701788387 71491633 298721145 1639185513 1642388068 288026373 1278690973 2000748657 1520068256 2065127301 974825955
100
1452886903 261229683 1649650969 615445921 1510450332 1424843593 1117310097 1621950351 1455031409 512646099 2011044530 1706542579 1711406731 334927656 1414135106 1303317829 192816720 711641346 1890227954 1399322353 1166192802 799280614 1585515253 1792083412 132295938 132433906 1230981416 1060587838 218596592 1860453403 934105971 972634768 775704731 434892875 1569313214 1222620190 2011732864 362861625 1909976358 416378282 382915295 21573864 673958322 207892143 69061234 1673925074 870898616 1409736453 1381114626 916743904 1147856204 73985247 1017765696 661649145 1711229111 1787134329 1893643094 384452114 880763698 1792496146 812537548 1914860524 1921655029 13023435 1024474305 497131572 730956984 1780935051 1878563033 1059881830 1708290674 101537555 1969182488 237819411 1996269254 529719853 1909499083 96721811 1715570727 669647936 1686453420 742952911 1976229999 1615838293 1291273108 2003783148 1917476343 447382245 281513912 121326772 342633424 805066505 1651598187 280005268 70541583 563881366 335154363 1791648530 319241286 428325690
100
920975798 1933090034 1799160693 1365472268 2082413951 1513699601 1346034180 291075839 1997729995 841949722 1685832642 759629619 1599842867 1892659421 185207090 202985944 827641246 225435183 1249136330 2088558339 868101232 1144837703 85418912 629917587 1662999236 1148916240 2065348381 1466635618 690643857 2127770459 891726426 1906098839 86837992 1435193267 216248780 733841440 400543195 1581037779 2005011331 1481920404 1550963677 1163707351 1439201958 1608251077 1301000466 1817054086 1233403478 890813486 1794757052 1294652266 223872347 1613868800 337684734 1098551688 1077658773 498540005 1329584069 999009335 191597429 790178907 961883826 1093949888 1871446618 139671555 2027754662 248234745 1829645066 786197266 347671120 1515065336 238765951 377502423 1987233487 868981124 732664686 1315356413 1087498871 1405798747 167870132 474673578 82743184 1984966606 296995434 7908627 833216406 2134352768 1647184319 1678518667 1179451233 610570882 1340543181 801112121 1373872287 1891518192 1738877089 549638154 725354620 1449496702 1756829178 1687545419
100
129740583 82355784 1294031374 712308835 1033908252 518683789 1242943316 1867293862 363620073 1411017945 576473128 108740074 176296388 818075723 1442892441 1646961654 1538813141 1252805418 381879314 860545660 1113035694 976160564 510650418 2092830975 1739401925 638168829 108510531 1777138528 1686162937 1043403259 702072619 818504005 1162237448 849204465 1562966660 1384814007 1209709319 462539259 660186340 568153419 93275252 1904446562 31350432 2068561533 442193894 779729438 1286950746 1693704728 740838753 691714958 442738311 762469054 240058093 889394778 1714829152 1739973320 95092622 1376394476 1969748234 1917993211 1166482100 1507164070 228704560 203023481 976782014 840591199 552676159 136782385 1838329674 167563817 259414727 604776158 875700313 240920523 1153313985 541709383 787908129 1093714264 792356681 1372292818 391330594 1190889228 1183948890 1147850463 360963244 2114111114 92587282 1962878268 936602297 1896635227 1082489295 1056167762 2052172832 1636404867 2097749584 1536884668 536625834 949354179 2016696428 1827631772
100
819020966 1096980625 722930449 329685827 1148812645 1134598933 1334530650 291881469 563187715 1277688947 574317510 1068020309 896798272 856471982 932588341 1441543523 1250519692 1546589152 122735187 659757922 1845664668 2120666660 1741814929 832928511 2059312389 1334917605 1059409253 1122325962 917997640 2007313157 982394002 255217471 1997073820 1257391014 1103483023 1830581681 1988377747 90211678 1719341655 1854322330 1227396205 1048486912 111109442 1042598616 1780388652 492085667 452521095 991048148 1122956696 1436590528 1808091936 2095462791 1708966128 1562829482 1247719678 21918678 164474968 1821442999 1259367975 2139874764 1802282151 628051346 1536113112 749426856 1324235314 673074034 539268579 1491300609 1318495135 466474369 113682986 598555798 879046355 1931464516 1676054181 1387529363 1993556865 1428962403 1212352348 1166095620 262278019 1990253957 355799984 1895745822 1338939511 1089325531 1955449592 1087122241 1696325248 788651276 1346010955 1700932560 559527767 272141221 65837354 1242184350 246407236 2062538979 1780718377 1028552902
100
907992491 880938977 1901807610 1846463691 289645203 746801264 1600093450 1343200234 1997387965 699643879 2124141741 16718807 2004056809 968713077 1578397186 2061498166 1949711279 798843098 327884548 1539114628 91078875 773187455 948668707 627524286 2082095673 803164695 964116437 1041664955 2033309904 165284339 198035642 130841139 1348038353 39763365 2069129050 220324726 821708147 1236238006 472705988 1607133447 1115913501 1613087232 880448075 1924980714 613015211 942931743 787163113 1309856448 1169395398 778715867 112678701 1118072623 211218777 954786470 1940537091 1056780850 683212292 7549989 1953097435 899271849 544722178 1870345685 1045954487 1970303227 840345911 915509039 1658717997 1913575945 1377797028 733642653 1348583608 719033870 1651902195 1348245353 879163973 102648976 1672440727 591896535 1309782463 431901698 1227039048 59726338 896728280 1263233588 1825269805 1445488306 1746870694 52380660 8440339 1349677828 617668294 1022415663 1551223004 734930574 288943729 594321476 1863162817 438686680 1513519017 481114573
100
481154651 1330838 6274364 899292002 1558388856 1356215061 267745260 545850650 430144841 165634381 1054609817 162650979 2006172609 1170560402 1241727053 401404044 1975819128 1424829863 1926167446 326257097 1575129984 1654357616 1966715198 44888173 1667885218 1271762328 1490329405 2118834674 1915697650 1502257695 1245556200 747956327 1827963951 1695829587 2006061927 1504861812 78831652 1248288482 545276242 1333814906 180737682 570155936 925793212 1035200666 2051411500 1154530800 753334271 1843205158 374860130 841141926 1305687121 1505025445 318683927 817059089 994996179 213018962 40323677 124741493 789738041 1980136803 903294487 724224701 610391228 1409655728 2063143184 1923371552 1302623363 1082369146 1346482659 1609071352 1789585397 1365997991 594031362 1812789851 220087617 836998224 1199877965 1155294528 1726774046 1348056941 1234515127 1332119642 2007398556 1568325499 821129293 2097187156 158746443 362854406 2081158125 203292303 371969593 344148214 1933831478 1257575434 1609294996 734186942 1040806586 1433082630 1780451919 210176069
100
874873808 251129418 790497026 519478710 228287214 135752951 744093606 666647117 1410349766 1354637311 1071121995 1028013775 2111721791 1197017014 895523696 425426219 1063783430 473764724 617211843 230939715 412689579 702712485 1024506421 791094321 794618435 1380305024 1059866866 1728616915 320381480 10628826 374659610 151906315 1174809515 843069382 1900168158 261660209 1462309188 233037354 1905350907 69034206 824885016 670279469 173920440 1684616779 711082728 1480528761 112228946 1236047380 705744183 923270283 182976547 249820745 1810496255 130890032 2114142493 156642186 1356284370 333989237 1272015677 1924703518 105488522 1851727187 860615254 83508203 1906098441 1804204694 1041064746 1694774014 731193887 1745931322 1979147399 1785018352 686419025 520084565 2082304018 620768988 1858611585 2053752686 1629847954 290495692 2092240879 1769320437 908163695 943094440 762461075 749362554 367398728 133349548 1766044705 1052620238 227052493 288726648 2048751078 2092377637 1407648810 739591409 789517859 1463339724 563788261 682259973
100
1385196931 1622391070 79257330 1187143591 929626427 1720960830 1392245355 1756624825 869899806 2018554629 847092271 1657550783 1577663086 764150992 577444467 1889825888 562561268 1946064223 903689910 999388393 466031882 303148750 1592760987 796412111 56125540 942332949 1105833269 715213278 199940232 1027766085 1053504633 378722177 583099569 1749493773 1109780550 656692508 813556823 2055705939 1315299113 1610274081 141125984 453502840 914346511 1457456097 244139480 108221033 1407460333 665294434 1150143140 2126969697 1754633185 1805333292 647616645 960406945 710392504 1185178251 1979140594 858492918 507241254 2020554526 371114763 1503651519 1888966474 1197339970 1941177390 201170389 228475616 1053473320 959777269 1672609909 910023881 476088711 1698864902 671452940 1263563543 460571977 1989574246 1937712930 549164070 1952431978 307937599 140587516 1550890810 2017799543 1802576929 1973154201 1424845977 1306328477 419985210 716785312 45138868 744371009 2015999956 880700674 681232408 410037144 2070939049 1481380639 1117053852 1390298270
100
792184238 1035110170 1435892898 1120487864 1445936807 2143322809 558272611 1847178687 945530274 156750600 1836066566 1897683960 241249072 1451486043 2807899 1527179545 1016465636 937158290 2049549917 1485427041 1107544911 1166317490 333158374 916864752 405645703 135238999 276160 383530894 678366829 958933421 164484700 1570003095 1259993312 423928192 68779770 47689773 1586533472 745792050 896537730 1598536626 1174272632 1462134237 867122031 1706158355 629147925 1123941654 64837443 1000378051 2094661441 697537793 1623794354 1234738957 276837692 303012884 182530241 97227088 357773840 7134133 219472114 1663360112 193832157 2145332080 1993806339 468978070 1133101087 1871406478 307984496 994512574 314174581 1345695484 1356047546 816201237 1528027593 982564695 1875495498 129487101 1939707848 803466901 1649742327 1966958713 983205638 447035919 1062612933 2045952089 706416665 1124599940 228608200 1423314917 763401093 842537292 1788729166 1967456768 1898470680 950627300 1541512198 1581109749 718283343 181958014 1789881454 1875615832
100
1718914583 1748345814 15844102 789570807 764598825 945029414 349844598 1901605698 788410480 1577164992 1545839016 2018097064 372703152 664697664 169618810 1017660912 2092199564 1292873907 1236283584 1769113243 1789138860 1226499260 1230597441 1748665034 2119796959 1785337142 1467579610 789852062 971087799 1240146591 1833091108 1837517439 1621072493 1054783436 1830508137 2137149222 534245628 1751463900 2146885719 386131464 114036751 858421051 1444844682 730816882 204695303 1114153411 512515230 1191857012 721002296 169584442 614319413 593417867 1999174018 988632645 697977599 132195485 1147633602 1703562235 1253464315 1230304253 1580335208 429023971 526192924 1859404807 1383706874 1428076987 32930434 104779411 634656624 1737655224 2093524578 571761219 179429759 337595182 2075726850 1868438976 2043936355 409894954 2102325758 1140279299 893060729 1983927267 142250312 1936244323 813359235 1617672313 974245527 214767208 1570805228 1602481995 283076231 916974943 67814089 589491963 769292877 696373882 1868240846 1047425228 846389969 1812521444
100
408904000 292311731 978832930 617012961 1637380329 1837882883 1833832711 1030351933 582162598 1638976274 1234426988 281237558 1258654057 850780349 245559057 1378344417 1163252030 2094090826 286558270 339292354 923370116 273482006 1008414230 1728245263 693361813 125597697 722040674 458586986 1034019743 1146664397 1399308358 1414216237 1321479175 356865100 2118462929 2063216226 505540102 333307163 614301478 1257591618 2020275540 2143301388 1908086800 1437629528 447257044 1287433298 859075965 645052355 635065641 729220053 325079607 1117233674 469783502 489610052 1675343318 254297650 417787180 637938678 1071692751 1991538893 1616978465 81362104 865699923 2064588948 461443700 2047258524 667322337 1991530726 1698407095 1629200759 649172947 193006862 289473748 236703215 1369307704 1429380083 484029473 56001168 1007648943 2120665592 1396739520 44891124 1369260199 720472441 1759396039 1663654994 2065163761 1526134327 731735976 1846745584 750315192 1430857859 1543072281 2066568849 1260719588 1100906874 737268840 1439229827 1900167476 1372767043
100
2112191576 344619657 307910864 30154046 1692591207 1347033063 728673077 1365737548 483256493 652446615 1882472572 957583366 1384576914 720613384 1547489396 1447290181 1655466492 1085407716 991744308 464431263 728295737 869427864 658674173 2024723810 1297505067 1993219332 6899906 144527255 772737220 1349416977 1584144585 1975401441 290759313 459780015 47671225 435613255 8769542 1761121906 1007990104 91421826 107485057 2003734072 1941623522 679301606 677233090 1759354255 1203665619 1062828436 1728001361 591983402 1734388171 301590579 620882565 1876889579 898619851 39986935 2021401637 2105813996 2122834237 2055228823 1949162957 1754979114 1022999086 1531915371 750100688 1826052079 627882896 807817832 2067274394 681716968 694718291 594827977 1484255822 2121710389 1246772051 1764197567 1849579303 1152965874 2045703992 1654620840 1045134318 1420901679 913542555 1917008881 111362398 100282183 1074505134 1520523751 1205042068 670714665 262371346 303212065 1029548791 1417902891 1476214893 1981293826 1242893502 1340169308 1894915467 692919417
100
789451656 118384440 931947588 870095752 528248722 1753746709 439711075 767942813 874280699 1395989675 1167846890 1598210684 586870839 1433991108 2085801505 1363786238 618537798 1422709676 399599341 201171258 780377058 1902543524 724141413 592156725 2136287471 1271692048 1305662116 1802176135 1003322207 2142451101 1348701771 1315589316 327926161 1731580020 1425753733 1888079737 1185404530 847770984 1743362269 1086862945 960453827 1817069290 1058379186 106298134 851428397 197240531 845491534 902344607 369711996 1001964487 829873734 74146985 561335077 1717241740 783285904 1036167095 827876313 164811389 866636372 1575649073 1991216235 759442891 1637702677 1594202056 2129207845 375160125 222089936 63637255 1844173509 622370691 557122629 1655077677 617180855 187429081 1297269135 479121065 2058061918 1215798015 671403765 808578300 1705969798 1408537499 269221877 1306431655 668281830 1603092886 940721278 476974735 2082524378 411865245 1420749838 992773891 1960138808 942733219 1544874011 196460295 107024024 482732017 1996469924 1239812075
100
536473535 1935934432 1477344226 272654225 266583070 1159780853 235763358 696013067 620049828 1020716654 997663452 465930943 747701821 913877840 1395533202 1447257493 1513161454 1756925428 1431063499 2090374011 1469599732 1177748578 2086250958 1211669268 1371298231 1617656531 619689302 1582996271 1984033376 1207423890 907854165 959658188 298753115 1420821235 1272099980 687417883 1465094662 677304837 1068173773 146232027 1929186078 828615877 247239011 1089359030 527174724 893313561 781660070 1656454541 773589174 669173832 858513327 955773192 1061295464 1231606163 1405919728 417442987 1244031609 482994876 1620305261 1247590760 1393469056 371958852 716172773 972978907 1607423659 367636188 1050576134 243662974 20035471 730116842 1496048087 1158592993 952290790 666143577 395217560 2116321549 867856810 1034405458 1534408744 1297307500 882358633 357456182 1458620232 696262313 1076817593 1057861491 482628868 1022953822 501890768 1284738332 2039860918 797486231 1898282890 282789330 1809593295 225710855 1140384929 726533783 1547184600 242985001
100
1156349554 1861405623 1853953911 598966305 1270712082 1970304657 1228011103 831886387 259909025 2062038753 1969433729 135982731 1284358100 1352370828 99930731 638453217 267492166 1795956961 926057731 2072491435 740288582 1373444036 420747633 1893874178 743918644 718530731 1642039895 48977755 1819006240 377416627 744694543 2023373797 2081699878 458905383 491287184 739902293 1011130421 821576273 862710997 353636588 896926748 2003466273 508112677 355785830 1728207782 202959967 1073244738 507222573 286648828 581560127 1651065674 1296266030 1747236176 1494304436 1821733148 1149197026 629566028 1798467628 205325398 666336033 838421499 1964577808 1013150883 1545345405 1070290621 1274831257 1105126062 1345709181 1526495623 1791401989 1302760725 1518220242 1222454817 377894988 1661038435 170429769 730165153 1966302796 558502561 2105680438 2082220519 645076042 1821891513 421576557 1260670007 2038593766 183512142 1672883072 1156469915 610156449 113334675 628029694 1493617510 990529852 1865289433 1398380144 1184464758 1312765752 352727577 379588806
100
602345168 428915676 2118435755 1090927628 1166418999 1299665433 1442995301 2040220231 218793971 113373258 1645704989 46646306 1658774763 1703661554 2038685722 1210244075 1357898973 880071233 1400279142 977411805 1511103372 1630171907 272480559 1528511079 578409625 660682491 1171342481 423488623 389903679 406849014 956582205 1496169302 1672234933 834551213 459160862 2069996803 439223348 1666176818 1695731513 333564292 1352810540 485579987 155284773 914994334 1872783857 1902493808 1749382830 603607637 1769411545 811456103 1836043875 1893035638 48083380 1651572081 1435158130 2013595559 418797012 2061845894 1166527799 1683116865 1816673098 1659433359 2090746263 508515420 1699444435 1559176798 2060305246 1743559611 39869769 1966546630 695982746 999990394 793236224 2009854792 1570755410 930064169 1253722871 783057354 514079443 533318707 976033171 2122326033 1409697253 566180013 1731874266 1402787561 1129090797 1398074927 632860562 2137807794 1577624281 777952095 1077168688 2126023842 2066122862 966183526 1955796571 294057549 531964796 1400192138
100
1877109670 663131319 563534012 542521062 348338309 1900923527 428172441 703485186 869763069 1703315532 1687342026 1636497262 117079142 1384737682 173274934 2090607951 1061919306 2069060625 706159268 1064950563 822384148 563300937 1080913255 1603054864 865379855 1804861267 1496905634 2082671510 332080702 855581723 1547831235 1601035794 52514430 357213958 120825441 2007120739 69653835 1596981548 768629767 957866193 128006893 106394838 1283214963 884226716 1058042345 61908094 1299071757 2086442241 368373816 152687002 2091730122 495970428 1813168233 1639776124 1514520322 565264505 792324014 1625572509 48548704 29858588 486227284 1375630369 974827043 1842647871 1593021957 212539147 1135499616 547497274 1551021166 631813297 1693099445 1224922626 204051873 810439206 903814414 1791346454 247785701 2068532547 502430603 943334263 2021936417 1685829789 1711351803 954879712 1099830839 456194651 1353380900 947587205 1359177442 2129588318 307451574 233557952 4517443 270582667 1395323629 1806703305 1273767598 504471886 1786137835 430932847
100
1169378064 10410244 267741238 1273173301 243534435 1870108586 500447319 599205983 402129666 1325050750 359549875 430547021 934575108 1446140057 500775591 26991156 449851343 1982528615 2087462252 1033851157 279585549 382041569 1741711826 719457095 35824683 1760517834 177472207 1023216560 973022995 284880691 1136359560 1024892116 1138524650 2092232151 550148040 86511168 1740400572 890593341 1910686700 1833684561 1835992925 1454046105 2087552278 194610324 90157200 721499290 1506138156 89766864 1701698801 1395248094 1852900137 1085528321 1255804315 592701493 661144806 1858722044 1785550038 418408708 1778061327 2043633915 1663027616 1938268714 2070269960 301593418 1357148092 122463484 1082000406 1942692144 107584591 1672398703 1903035325 218226054 2095794817 1245092191 2087022115 1372640755 670634675 956927483 700457918 1144569108 1900710814 653226593 209367572 110104754 58423769 1606736753 386969500 2121013400 1899856689 1175199661 1514439394 574218479 478876473 2098949008 286990822 339964361 1952511613 1532842860 1667924914 721538413
100
1579708984 780890151 999152175 257328137 1855406542 446996410 216239812 1766785935 256517827 819110249 1385934 1927370937 408488235 1479764105 1743161584 1629146330 54831071 1458891234 1118486022 1827049875 1630451954 1236827504 1402650699 1686307114 849794804 1425220558 1125177519 1744604373 1774788303 340004006 904827259 1262954381 1973413168 1700688394 1887186743 1529038784 1559816839 1372203781 1913114512 1115248193 435884806 1476909714 1379496396 333352428 1049715036 3357454 97488854 64452976 924779065 549115412 1922587153 1796137487 1451392521 16122420 702245709 218096337 34589160 1626846283 920013608 941222085 530829025 121509645 1820312264 557269731 1283797455 1787853726 122843519 225041974 1240487732 874058349 1768572682 210014542 948746249 1024293225 213297117 1195720817 1174121960 315617687 1165742563 1926088967 502265528 149661260 637962575 1187160711 1424212661 1444478365 1357327117 83662903 101339954 905838043 537135455 123747355 528650900 899287524 279916763 69920513 1496610299 2113821960 1586110861 667269942
100
728214174 914052903 457954967 271955065 1581193032 287607223 1986018691 469823253 2098695339 1546511603 489666977 1660328914 2022979078 1702662125 43465923 283883513 684287495 137516113 2016407775 1818169302 1449745667 1030013088 1404658731 754665331 1210479619 1274280025 1591747345 1484560737 1415769821 1466745356 1589336305 871520611 1362611151 1699359722 1254625394 1851908851 1804972426 1310476930 772689796 1069161557 1574504817 466756761 688338030 468214110 1091379181 2108936632 1273242106 2016133022 1441416386 1129786017 370779806 25400354 1073103823 1537673974 45008701 1526682586 1299819949 874548878 668170443 88452817 1920523893 2051149278 1576398751 303729704 864699736 1657894036 2099722772 386068023 654626571 839855635 1810556341 1190509251 1705135746 754743231 1981232523 952028607 536765453 1713488066 1817071248 543810157 351405979 1880755134 1125907582 766939057 1639705612 1659192251 938816391 807543001 1832960318 1706328305 694619698 1650974727 1070671432 210864245 974770320 1019503774 1096435260 53842705 862311545 206882531
100
1138079613 1550250140 1533140852 1822103607 649945577 969486004 1999937649 7150523 148972394 113778635 375141912 1450847008 1652448592 1105558430 1105332979 414534430 77507320 1291132477 1749495710 462178351 714258121 1444683628 1506960839 668270181 1459299782 1441778060 1805653611 595531291 298171374 986163055 913506139 1015830830 872817166 1049742297 1106310118 443634465 579677568 531026043 1410340683 1650601007 624988754 2045549292 1895416494 1116888740 1427342754 1472208795 342197989 77441042 422812534 690388370 462040215 485413864 255427559 1365153849 994165922 522618105 1152942765 937877458 1414367618 284780449 1342241214 1140570595 142474508 154343248 675859001 692855064 2116640424 2014160497 1754778938 1002947956 1513648513 429955948 1862282651 1680270775 631903395 524944147 2004057247 1111021829 1592767074 1621245421 943704016 1032779511 909436521 453187793 1141480131 1081919129 432194134 1160686880 818982117 1015456691 1584404227 1018379909 1822041373 1024729221 119772430 720005085 158574551 1179015974 1709025568 1548501066
100
272312050 1331289510 874629786 1466080517 799558881 1892457685 1981122949 642616762 2125047293 42386393 98096812 1436246976 1183681515 1562761224 793075643 1064043324 701714986 1007784627 855086311 2135919143 245091767 1723685867 1288983680 256844364 798097612 695187241 313178109 1561185685 197354366 1033562829 81822220 1737247047 1561407713 1979361062 1343675007 2053017324 963732799 266745738 53882980 1989418490 364858264 1644635012 137226312 107441700 464683110 1026480605 1638369007 1240447655 1262540656 1883722059 1649316886 1640199858 1481993679 708284350 1696205078 1875457965 1708529722 1502585315 1101447762 614840300 64043510 1176899888 511096118 1940465772 370709397 407033482 614744361 1391870051 1747715053 1041307547 1771861447 1719313902 1036654219 1810456252 613228465 1826955923 2001979745 727244866 53946552 529223145 1202199429 39860231 1262865785 200899229 598669979 1915073914 906272044 43096553 324597896 1927816274 187983022 809127235 671245301 1928682418 1380020650 747060682 1590939568 1068494421 1704823554 1998112654
100
561075597 938186315 310894803 691400184 612159204 718545790 2075555514 1718132099 869346344 1713098838 1232525140 965730982 392056765 1233736985 1554532548 1885848664 1594233296 783863698 973280119 1540293349 1118367202 1202663049 695014029 567091826 1158483773 2117295133 1076644687 1366720330 1687729837 770096192 440911536 93515691 1290665019 418059042 1309964770 1800118515 666372401 390081454 2028624815 409876801 1994400829 690555686 20704421 245107423 1544356408 1519792447 1025557694 178043016 1801197467 1784286969 1487613909 744971581 1541778765 1951630795 1585669658 1838755064 1975842859 1854953474 1008503685 870387439 361346512 27988164 435356655 2129009171 1581635516 1973312481 1126986247 691786752 947803279 1217085386 869385484 1182122282 439459011 1435362262 1558693334 642415629 1434924984 175245498 896754596 800105065 880594500 500460248 609858305 68966996 431597110 2051510382 1754878522 150067575 1810720569 367588740 1308993953 1599038486 1273464695 1801780511 1984833694 579846440 390776928 262067188 1975424315 1060670787
100
1622634552 461662602 1072312650 155131665 840694593 555463458 1700173309 516977988 916317474 965289368 1410019355 1946010302 1694220598 1081396601 1989316183 1175129807 1468475511 483101629 1026642372 875430255 1986444183 939582843 190415616 1466033877 1623885850 2007171136 2126059741 1471182841 424228550 352076793 455779391 1688247734 890035250 1107943738 179249807 1512653145 1489617637 993027 177466802 1094303449 927367800 2012030734 298402103 1118338042 693196428 1684023820 800935128 1723263425 675773094 356896315 1296096644 1110348564 514097260 1769730194 613405309 625909074 1213368216 1466995424 746998923 1110972689 466936521 1197528959 1918447188 620509627 542933258 1100010266 1070628424 72003300 2056634642 765106929 133347204 1402687277 1926577717 31543248 251152487 762187043 868536645 1756980602 784181132 149156325 1586048434 422431532 1568462892 1400106498 1667587130 1040337807 2124972347 1836509147 681532497 1797067573 1466697481 520628494 831448859 427540954 1446568553 1934878080 1271452639 623855720 328014342 577275615
100
719549621 1866910557 818034099 1500862278 2129355608 146991548 1273515999 753967939 704085814 2113717403 229990219 495199908 924693869 1750679219 1855627404 1234531034 1187491456 988230032 1506018114 2101048121 415497052 1169797391 916552530 963132253 1544827131 1430991124 504310482 1733188715 878526022 803126637 1978898983 132318614 663417 1381832624 151449711 662838919 966325410 685065581 670979737 588264907 630002279 1840245671 410864592 1571509213 2022511877 1815711142 2112148264 2036144823 673963637 1403992379 493754976 4292071 1013529127 921951131 884272458 1067657670 563856144 879341774 2083385774 25236067 839983131 342670697 135590141 468638743 1879759443 1768637970 994135190 14681418 1689881927 1107323199 1787287715 1350884357 734595467 967612531 1091610208 1527733180 470059385 1044803963 764051501 1563695779 1130938884 1261455432 611282320 351702847 1346105776 1803422739 2015725452 1084781346 165748015 1895042059 1090465413 830998303 655214601 1354109923 1803449877 684423566 238394390 1806847014 906524640 1419960897
100
1988236745 1213868736 848309628 1099853775 501581753 1027360803 827730309 1899379717 10026332 1791293999 1688166086 634687951 250662287 788157221 1298014793 354423184 1956164978 2074807242 2069572208 1762352168 1873856424 1856894710 1248802186 522032505 143703217 881021429 1020414001 979318045 1022527525 1804025236 1227060938 57409639 1499955520 580425381 652142887 1657142237 1347014812 277143840 82974794 1936299974 1415892661 1931180629 280169239 408168975 2068930105 410737236 1675555535 667369148 891936729 1116390794 1374032496 356527424 881585041 1350334683 1077766465 929070575 1657672929 160675122 802978513 1769738483 878675683 595202115 1116697816 846174914 397911851 1620061304 315133319 1343278079 385830622 1070942853 1409050605 966667051 734592893 811154814 1560352839 1289760635 1493734022 734169802 1980151871 8070862 2085836407 2060416076 1656969358 1357128273 1101199005 1297471865 1493742684 119609040 1741848176 1927719236 1388146370 117550956 1718217033 1297864518 698650094 871844809 1935954515 146557808 1689183618 219150805
100
725906923 240023181 1416698715 1446614758 696957368 2023986993 63318221 1486469431 406827360 1085205775 1926392599 1967061590 67676069 256750053 1462410560 1517132489 655001222 1487010195 95287467 763047406 1158556356 1446483698 1054168641 654010007 215688977 2008369538 1589050771 312035813 532630118 808470650 2130901140 860212478 331684012 363046477 624337664 16446271 854214051 177534626 979594991 1452312482 1455663903 1111088259 1569176343 1532238756 1685604512 1298926275 174441493 1987342727 1002647974 582822452 1836464803 271934812 424220779 868203593 134436465 83842087 2026718358 715815442 1996113396 114200463 1453764471 1000913539 1879838928 1065718450 629522229 1957808576 666040153 576623945 589074058 875557436 690624227 2085618091 1947456736 263736181 1906689596 613623943 1742039780 897937583 1954609135 508130132 888288781 1816828659 1854082415 703667316 1187519783 835480802 445389467 1200214419 2051892878 574430921 1784089438 353784706 144078058 996934740 1578770337 729392408 2012156715 1521004713 1241047398 216955658
100
2052031383 761597139 1607873757 970349546 1679725333 498285732 395927667 89470769 43040063 195779777 2094662123 692517361 1686907863 287716150 1906808861 1103928610 724383654 516700833 1949684584 1326509241 66219759 892201610 956294092 1273990706 1931585372 380569354 990716177 68281005 307250954 2071318307 1654915780 725667668 674805454 83882826 889157537 997867744 1387303129 1942482744 131289784 780946778 1037506430 291821093 162721273 1781234514 400467001 1272967439 758500057 232792087 536883651 1842529158 1807813491 2074852929 892730966 732497207 166058825 494348169 1426540089 1457951017 1025785708 1517354979 1747604095 622319569 1777835905 1426522111 687855081 145506251 446263353 1186709828 206197154 706672517 1814522236 404474080 390510902 1260953626 1072404352 421174966 1755448027 767133218 1035418738 2124587074 1495548396 393091511 1528624940 963874843 1811557757 1537153797 917524280 1900248670 215701759 1546235863 1458648438 732282853 2107892222 1350196798 1330252184 1541484589 1018762314 65142314 484236213 386189868
100
1125312882 1845649267 1489872838 2116869911 802221957 1156715652 1223012326 1010647212 1560680770 1898344681 1870280011 1119400083 194792775 1774441302 1830799511 2111648117 1008553536 298693141 699775430 1526314336 408546189 2028860463 1964376678 2106573840 422851448 139853333 669046203 933268552 1757892022 675564679 879284730 883064373 875340274 1458832713 1067729265 1345591072 1556742801 341217861 424357114 1370112556 1675279455 1602324928 1570447812 686766042 29567855 2123728171 1198946890 1825198394 1751348459 36285293 1222901359 1227761276 1011672099 1641066419 1474432848 1096461803 926051569 12571759 542858316 1533406429 1064589441 1848804492 636775906 1512050268 1571921251 1242549441 41769677 364201823 1252614611 267023148 200244383 508900886 125651174 441460736 790600217 2064601927 964974108 1589826530 1100940608 1011373379 471413383 1144464933 1628183646 875413756 719860123 1515385287 1535032079 856348461 2086829826 1704978980 729313028 1099074367 1144511482 1341605755 278994539 1844947134 1716337190 1475650274 2143771386 1626380538
