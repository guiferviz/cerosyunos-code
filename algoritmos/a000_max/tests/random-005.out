2145717105
2011732864
2134352768
2114111114
2139874764
2124141741
2118834674
2114142493
2126969697
2145332080
2146885719
2143301388
2122834237
2142451101
2116321549
2105680438
2137807794
2129588318
2121013400
2113821960
2108936632
2116640424
2135919143
2129009171
2126059741
2129355608
2085836407
2130901140
2124587074
2143771386
